supporting_content/dimensions/region.md

```diff
@@ -0,0 +1,69 @@
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider | Data set                 | Column            |
+|----------|--------------------------|-------------------|
+| AWS      | CUR                      | Product_region    |
+| GCP      | Big Query Billing Export | Location.location |
+| Azure    | EA, MCA                  | ResourceLocation  |
+
+## Example usage scenarios
+
+Current values observed in billing data for various scenarios:
+
+| Provider | Data set                 | Example Value                                                                   |
+|----------|--------------------------|---------------------------------------------------------------------------------|
+| AWS      | CUR                      | us-east-1, eu-west-1, us-gov-east-1                                          |
+| GCP      | Big Query Billing Export | northamerica-northeast1, asia-southeast1, us-central2, europe-west3, us |
+| Azure    | Cost details             | Eastasia, AP East, WestUS, centralindia                                   |
```

Comment:
Needs to be changed across the board from Azure to Microsoft  might be worth someone doing a global find and replace to ensure consistency
```suggestion
| Microsoft  | Cost details             | Eastasia, AP East, WestUS, centralindia                                   |
```